### PR TITLE
Move from ex typesense

### PR DESCRIPTION
**Pull request by jaeyson**
To decode a [`JSONL` format](http://jsonlines.org) when [importing multiple documents with newline-separated JSON](https://typesense.org/docs/27.1/api/documents.html#import-a-jsonl-file), here's a tip from [`Json` library author](https://github.com/michalmuskala/jason/issues/34#issuecomment-374210513):

> I think this could be also solved with something like:
> 
> ```elixir
> jsonl
> |> String.splitter("\n")
> |> Stream.map(&Jason.decode!/1)
> ...
> ```

### CODE REVIEW

**Comment by kianmeng** [comment]
@jaeyson Shouldn't be open_api_typesense just simply the generated api wrapper for Typesense and ex_typesense is custom wrapper with Ecto over open_api_typesense?

**Comment by jaeyson** [comment]
> @jaeyson Shouldn't be open_api_typesense just simply the generated api wrapper for Typesense and ex_typesense is custom wrapper with Ecto over open_api_typesense?

Yes, open_api_typesense is a wrapper. The library is heavily WIP as of now and please do correct me with the following:
- http client is currently using `Req`. will add different adapters in the future.
- open_api_typesense has no knowledge whatsoever with Ecto, based on the PR.
- the results of open_api_typesense is mostly `{:ok, Schema.t()`, `:error` or `{:error, message}` as basis from the generated `oapi_generator`
- ex_typesense will depend on that library and Ecto.
- tests in ex_typesense consists of common usage of the library + Ecto
- tests in open_api_typesense only revolves in the implementation, no Ecto.

**Comment by jaeyson** [comment]
Quick question @kianmeng: have you encounter an error like this?
https://github.com/jaeyson/open_api_typesense/actions/runs/12153877902/job/33892694299#step:14:11

I comment out lines for `mix format` as I keep seeing this weird errors in Github Action where I already formatted all the files before commit + push.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+          elixir: '1.17'
+          lint: true
+        - typesense: '26.0'
+          otp: '27'
+          elixir: '1.17'
+          lint: true
+        - typesense: '27.1'
+          otp: '27'
+          elixir: '1.17'
+          lint: true
```
Comment (kianmeng) [inline .github/workflows/ci.yml:54]:
```suggestion
        - typesense: '0.25.2'
          otp: '25'
          elixir: '1.14'
          lint: false
        - typesense: '26.0'
          otp: '25'
          elixir: '1.14'
          lint: false
        - typesense: '27.1'
          otp: '25'
          elixir: '1.14'
          lint: false
        - typesense: '0.25.2'
          otp: '27'
          elixir: '1.17'
          lint: false
        - typesense: '26.0'
          otp: '27'
          elixir: '1.17'
          lint: false
        - typesense: '27.1'
          otp: '27'
          elixir: '1.17'
          lint: true
```

@jaeyson You shouldn't set lint to all matrix combination, only for `ONE` combination of latest greatest Elixir/Erlang OTP version. Perhaps this explains the `mix format` error you're facing?

Comment (jaeyson) [reply .github/workflows/ci.yml]: @kianmeng wow that helped! Thanks alot! 🙏🏾🙏🏾

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**